testplan/runners/pools/base.py

```diff
@@ -491,7 +473,7 @@ def _handle_taskpull_request(
                         continue
                     else:
                         if self._can_assign_task_to_worker(task, worker):
-                            self.logger.test_info(
+                            self.logger.user_info(
                                 "Scheduling {} to {}{}".format(
```

Comment:
This is not in C-type string format.
Alongside with line 430, 492, 537, 554, 585, etc...